### ISSUE TITLE
Add devops

### ISSUE THREAD

**Issue by kalexmills** [github-vet/bots#15]
There is no reason I should be running these bots from my local machine -- even though it works (for now), there are obvious issues involved.

Ideally both `vet-bot` and `track-bot` would be deployed in the cloud. All it requires is one container and a persistent disk (for recovery).

If anyone wants to take a stab at that, I'm open to PRs. Otherwise I'll get around to it once testing is done.

Also want CI/CD, possibly via Travis. Any expertise there would be great.

**Comment by kalexmills** [comment]
This was partially completed via #38. What is needed now is much more specific, so I'm closing this in favor of another issue.
